src/main/scala/is/hail/variant/GenomeReference.scala

```diff
@@ -8,8 +8,38 @@ import is.hail.utils._
 import org.json4s._
 import org.json4s.jackson.JsonMethods
 
+abstract class GRBase extends Serializable {
+  def isValidContig(contig: String): Boolean = ???
```

Comment:
I think it's cleaner to put ??? implementations on GRVariable and have these abstract with no overrides.